updateHostsFile.py

```diff
@@ -1063,6 +1063,15 @@ def flush_dns_cache():
                 else:
                     print_success(nscd_msg.format(result="succeeded"))
 
+        centos_file = "/etc/init.d/network"
```

Comment:
May I ask if we are sure that this file is only present into CentOS ?

Reply:
I'm not aware of any other distros at the moment.

Reply:
`Redhat`, `Fedora`, `Suse` and `OpenSuse Linux` ?

 I'm pretty sure there was (and maybe still is) a `/etc/init.d/network` into those distros.

Reply:
Ah, that's right.  In the docs, it's referred to as `RHEL / Centos` (though obviously such a variable name isn't possible in Python).